src/pipeline/fit/FIT.groovy

```diff
@@ -0,0 +1,58 @@
+package pipeline.fit
+
+def configControlInterfaceIp(String rackhd_dir){
```

Comment:
@changev  @kelly , please carefully read this file , which will impact your ova-post-test